arch/arc/Kconfig

```diff
@@ -328,4 +328,11 @@ config ARC_EXCEPTION_DEBUG
 	  and parameters, at a cost of code/data size for the human-readable
 	  strings.
 
+config ARC_CUSTOM_INIT
+	bool "Enable custom initialization code"
+	default n
```

Comment:
Default n is not needed, here (it is default n, anyways), so it should be removed